jinahub/indexers/storage/MongoDBStorage/manifest.yml

```diff
@@ -3,7 +3,7 @@ name: MongoDBStorage
 alias: MongoDBStorage
 description: Storage-type indexer powered by MongoDB
 author: Jina AI Dev-Team (dev-team@jina.ai)
-documentation: https://github.com/jina-ai/executor-indexers/tree/main/jinahub/indexers/indexer/MongoDBStorage
+documentation: https://github.com/jina-ai/executors/tree/main/jinahub/indexers/indexer/MongoDBStorage
```

Comment:
this is a wrong URL. 404

Reply:
after merge it will be available

Reply:
No, because the folder is `storage/MongoDBStorage`

Reply:
oh okay good catch